modules/aleph/src/Aleph/AlephDebt.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+use Drupal\aleph\Aleph\AlephDebtMaterial;
+
+/**
+ * Class AlephDebt.
+ *
+ * Implements a class with a patron's debt.
+ *
+ * @package Drupal\aleph\Aleph
+ */
+class AlephDebt {
+  protected $type;
+  protected $description;
+  protected $sum;
+  protected $date;
+  protected $debtMaterial;
+
+  /**
+   * Set the debt type.
+   *
+   * @param string $type
+   *    The debt type.
+   */
+  public function setType($type) {
+    $this->type = $type;
+  }
+
+  /**
+   * Get the debt type.
+   *
+   * @return string
+   *    The type of debt.
+   */
+  public function getType() {
+    return $this->type;
+  }
+
+  /**
+   * Set the description of the cash transaction.
+   *
+   * @param string $description
+   *    Description of the cash transaction.
+   */
+  public function setDescription($description) {
+    $this->description = $description;
+  }
+
+  /**
+   * Get the description of the cash transaction.
+   *
+   * @return string
+   *    Description of the cash transaction.
+   */
+  public function getDescription() {
+    return $this->description;
+  }
+
+  /**
+   * Set the debt sum.
+   *
+   * @param string $sum
+   *    The patron's debt sum.
+   */
+  public function setSum($sum) {
+    $this->sum = $sum;
+  }
+
+  /**
+   * Get the debt sum.
+   *
+   * @return string
+   *    The patron's debt sum.
+   */
+  public function getSum() {
+    return str_replace(array('(', ')'), '', $this->sum);
+  }
+
+  /**
+   * Set the cash transaction date.
+   *
+   * @param string $date
+   *    The cash transaction date.
+   */
+  public function setDate($date) {
+    $this->date = $date;
+  }
+
+  /**
+   * Get the cash transaction date.
+   *
+   * @return string
+   *    The cash transaction date.
+   */
+  public function getDate() {
+    return $this->date;
+  }
+
+  /**
+   * Set the debt related material.
+   *
+   * @param AlephDebtMaterial $material
+   *    The material which is the reason for the debt.
+   */
+  public function setDebtMaterial(AlephDebtMaterial $material) {
+    $this->debtMaterial = $material;
+  }
+
+  /**
+   * Get the debt related material.
+   *
+   * @return AlephDebtMaterial
+   *    The material which is the reason for the debt.
+   */
+  public function getDebtMaterial() {
+    return $this->debtMaterial;
+  }
+
+  /**
+   * Returns the patron's debts from SimpleXMLElement.
+   *
+   * @param \SimpleXMLElement $xml
+   *    The SimpleXMLElement from the Aleph Cash API.
+   *
+   * @return array
```

Comment:
Actually ```\Drupal\aleph\Aleph\AlephDebt[]``` - they do have an AlephDebtMaterial but this is set on a property on the AlephDebt objects.